query-node/mappings/src/membership.ts

```diff
@@ -187,23 +194,25 @@ async function createNewMemberFromParams(
     id: memberId.toString(),
     rootAccount: rootAccount.toString(),
     controllerAccount: controllerAccount.toString(),
-    handle: handle.unwrap().toHuman()?.toString(),
+    handle: ('unwrap' in handle ? handle.unwrap() : handle).toHuman()?.toString(),
     metadata: await saveMembershipMetadata(store, undefined, metadata),
     entry: entryMethod,
     referredBy:
       entryMethod.isTypeOf === 'MembershipEntryPaid' && (params as BuyMembershipParameters).referrerId.isSome
         ? new Membership({ id: (params as BuyMembershipParameters).referrerId.unwrap().toString() })
         : undefined,
     isVerified: false,
```

Comment:
In runtime [implementation](https://github.com/thesan/joystream/blob/9ffc1bc4a9a771be67975edf1702f2399507472c/runtime-modules/membership/src/lib.rs#L1219-L1220) of the extrinsic, the founding member is can set to verified status, however in mappings its false for all cases. 
```suggestion
    isVerified:  entryMethod.isTypeOf === 'MembershipEntryFoundingMemberCreated',,
```

Reply:
Thank you I had completely missed that